flexmeasures/cli/data_add.py

```diff
@@ -1103,6 +1119,8 @@ def add_schedule_for_storage(
             "consumption-price-sensor": consumption_price_sensor.id,
             "production-price-sensor": production_price_sensor.id,
             "inflexible-device-sensors": [s.id for s in inflexible_device_sensors],
+            "consumption-price-sensors-per-device": {(power.id, price.id) for (power, price) in consumption_price_sensors_per_device.items()},
+            "production-price-sensors-per-device": {(power.id, price.id) for (power, price) in production_price_sensors_per_device.items()},
```

Comment:
As the schema expects a dict, wouldn't it be better to pass the dicts here as-is?

Reply:
@Srieon @rajath-09, this is something we could do ...

Reply:
> As the schema expects a dict, wouldn't it be better to pass the dicts here as-is?

Addressed.